osf/models/node.py

```diff
@@ -356,7 +356,7 @@ def parent_node(self):
         return None
 
     @property
-    def quick_parent_id(self):
+    def get_parent_id(self):
```

Comment:
@sloria, this would require a broader change to `api/base/serializers` since `get_parent_id` is being used on the relationship field to build the parent url and is expecting an attribute there.   My opinion is that we should leave a property and change the name back to something that doesn't have a verb.

Reply:
Ah ok. How about making this the implementation of `parent_id` (already exists) and rename `get_tags` to `tag_names`?